aspnetcore/fundamentals/environments.md

```diff
@@ -69,6 +69,48 @@ The `Production` environment is the environment in which the application runs wh
 
 This is by no means meant to be a complete list. It's best to avoid scattering environment checks in many parts of your application. Instead, the recommended approach is to perform such checks within the application's `Startup` class(es) wherever possible
 
+## Setting the environment using the command line
+
+Setting the environment can be set from the command line when starting the app. Depending on the operating system, the method in which this is accomplished varies for either a global or session based setting.
+
+### Windows
+To set the `ASPNETCORE_ENVIRONMENT` for the current session, if the app is started using `dotnet run`, the following commands are used
+
+**Command line**
+```
+setx ASPNETCORE_ENVIRONMENT "Development"
+```
+**PowerShell**
+```
+$Env:ASPNETCORE_ENVIRONMENT = "Development"
+```
+
+These commands will last for the duration the current window is open, once closed the `ASPNETCORE_ENVIRONMENT` setting will revert to the default setting or machine value. In order to set the value globally on Windows open the *Advanced System Properties* > *Environment Variables* and add or edit the `ASPNETCORE_ENVIRONMENT` value.
```

Comment:
> These commands will last for the duration the current window is open, once closed the `ASPNETCORE_ENVIRONMENT` setting will revert to the default setting or machine value.

Suggested rewrite to avoid run-on sentence:

These commands take effect only for the current window. When the window is closed, the `ASPNETCORE_ENVIRONMENT` setting reverts to the default setting or machine value.

UI elements should be bold.

Control Panel > System > Advanced system settings (not properties)